app/models/knight.rb

```diff
@@ -1,3 +1,17 @@
 class Knight < Piece
 
+	def is_valid?(new_x, new_y)
+		total_x = new_x - self.x_coordinate
+		total_x = total_x.abs
+		total_y = new_y - self.y_coordinate
+		total_y = total_y.abs
+		if total_x == 1 && total_y == 3

```

Comment:
I believe that the difference for the new move can only be 2 when the other axis change is 1.

Reply:
oh yeah, whoops. I'll fix that